target/linux/apm821xx/image/nand.mk

```diff
@@ -50,8 +50,9 @@ define Device/meraki_mx60
   KERNEL := kernel-bin | libdeflate-gzip | MuImage-initramfs gzip
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
   UBINIZE_OPTS := -E 5
-  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_VERSION := 3.0
```

Comment:
Please also update `target/linux/apm821xx/base-files/etc/uci-defaults/05_fix-compat-version` this breaks sysupgrade otherwise.